style-guide/extra.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
```

Comment:
Wondering if this should be broken up into individual files like the others.

(Maybe it doesn't matter if we're going to move to having the source be the style-guide file...)

Reply:
I actually created one file since all these checks are run directly without proselint. Proselint did not produce correct output with quoted text, so I had to create these separately. I kept them in a single file so that only one file is imported while checking. If we have separate files, we will need to import all of them in the style-test script.